system-tests/projects/cypress-in-cypress/cypress/e2e/123.spec.js

```diff
@@ -1,6 +1,6 @@
 describe('Blank Contents', () => {
   it('renders the blank page', () => {
-    cy.contains('cy.visit()')
+    cy.get('svg')
```

Comment:
can we add a id or data attribute to this svg so we can verify this a bit more?

Reply:
added [9bb31cf](https://github.com/cypress-io/cypress/pull/23095/commits/9bb31cf7ba7472b0b2371d1ea90ebbe8fbe93002)